_posts/2020-11-05-welcome-to-jekyll-demo-post.markdown

```diff
@@ -0,0 +1,156 @@
+```SAM
+1:497:R:-272+13M17D24M	113	1	497	37	37M	15	100338662	0	CGGGTCTGACCTGAGGAGAACTGTGCTCCGCCTTCAG	0;==-==9;>>>>>=>>>>>>>>>>>=>>>>>>>>>>	XT:A:U	NM:i:0	SM:i:37	AM:i:0	X0:i:1	X1:i:0	XM:i:0	XO:i:0	XG:i:0	MD:Z:37
+19:20389:F:275+18M2D19M	99	1	17644	0	37M	=	17919	314	TATGACTGCTAATAATACCTACACATGTTAGAACCAT	>>>>>>>>>>>>>>>>>>>><<>>><<>>4::>>:<9	RG:Z:UM0098:1	XT:A:R	NM:i:0	SM:i:0	AM:i:0	X0:i:4	X1:i:0	XM:i:0	XO:i:0	XG:i:0	MD:Z:37
+19:20389:F:275+18M2D19M	147	1	17919	0	18M2D19M	=	17644	-314	GTAGTACCAACTGTAAGTCCTTATCTTCATACTTTGT	;44999;499<8<8<<<8<<><<<<><7<;<<<>><<	XT:A:R	NM:i:2	SM:i:0	AM:i:0	X0:i:4	X1:i:0	XM:i:0	XO:i:1	XG:i:2	MD:Z:18^CA19
+9:21597+10M2I25M:R:-209	83	1	21678	0	8M2I27M	=	21469	-244	CACCACATCACATATACCAAGCCTGGCTGTGTCTTCT	<;9<<5><<<<><<<>><<><>><9>><>>>9>>><>	XT:A:R	NM:i:2	SM:i:0	AM:i:0	X0:i:5	X1:i:0	XM:i:0	XO:i:1	XG:i:2	MD:Z:35
+```
+The **body** consists of alignment reads which each have 11 parts. Some terminology to also keep in mind is:
+
+**Template**: The DNA fragment that was measured
+
+**Reads**: A template may produce one or more reads. These reads may cover just a subsection of the template.
+
+**Segments**: Each read may produce one or more alignments that will have aligned regions called segments.
+
+![SAM alignment parts](https://bioinformatics.uconn.edu/wp-content/uploads/sites/15/2017/01/format2.png)
+
+1. *QNAME*: Query template NAME. Reads/segments having identical QNAME are regarded to come from the same template. A QNAME ‘*’ indicates the information is unavailable.
+2. *FLAG*: Combination of bitwise FLAGs (more information below).
+3. *RNAME*: Name of reference sequence. It generally refers to chromosome number.
+4. *POS*: Leftmost mapping position of the first matching base in read. It has 1-based indexing. If pos is set as 0, it represents a unmapped read.
+5. *MAPQ*: MAPping Quality. It equals −10 log10 Pr{mapping position is wrong}, rounded to the nearest integer. A value 255 indicates that the mapping quality is not available.
+6. *CIGAR*: Concise Idiosyncratic Gapped Alignment Report (CIGAR) string. It describes alignment characteristics.
+7. *RNEXT*: RNEXT is the name of the chromosome or contig to which the next template in a pair aligns. RNEXT of value ‘=‘ means align to same reference and ‘*’ represent no information available (single end sequencing)
+8. *PNEXT*: Position of the primary alignment of the NEXT read in the template. Set as 0 when the information is unavailable. This field equals POS at the primary line of the next read.
+9. *TLEN*: Template LENgth. It is set as 0 for single-segment template or when the information is unavailable.
+10. *SEQ*: segment SEQuence.
+11. *QUAL*: PHRED score values of read. If ‘*’ no values are stored.
+
+# VCF
+Variant Calling Format/File. It is a text file that contains a **header** and data lines that constitue a **body**. It stores gene sequence variations.
```

Comment:
Nice. So I noticed that VCF and SAM are far more complex. Given my being new to the -omics, I could use a sentence for VCF and another for SAM just giving very rough context for why each is used and how they differ. I see that they both compare sequences... I think.